definitions/json/AuthorisationCaseField.json

```diff
@@ -21767,6 +21767,18 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
+    "CaseTypeId": "ET_EnglandWales_Listings",
```

Comment:
confused as to why these fields are being added to the Listings and Multiple caseTypes